parm/config/gfs/config.anal

```diff
@@ -148,6 +148,17 @@ if [[ ${RUN_ENVIR} == "emc" ]]; then
   #   fi
 fi
 
+if [[ ${USE_BUILD_GSINFO} == "YES" ]]; then
+   # these will be set based on date in scripts/exglobal_atmos_analysis.sh
+   export SATINFO="generate"
+   export CONVINFO="generate"
+   export OZINFO="generate"
+   export OBS_INPUT=${FIXgfs}/gsi/build_gsinfo/obs_input/obs_input_reanl_nasa_ozone.txt
+else
+   export OBS_INPUT=${FIXgfs}/gsi/build_gsinfo/obs_input/obs_input_ops.txt
+fi
+export HIRS_FIX=/gpfs/f6/drsa-precip4/proj-shared/Jeffrey.S.Whitaker/hirs_fix
```

Comment:
noting a hard coded Gaea path here

Reply:
Thanks @CoryMartin-NOAA - I think this needs to default to the CRTM_FIX directory and be configurable in config.anal.  We won't need any of this when CRTM is updated though (hopefully).

Reply:
The `hirs_fix` data has been added to the `build_gsinfo` repository, so there are no longer any C6 paths in config.anal.